sonar-plugin-api/src/main/java/org/sonar/api/rule/RuleKey.java

```diff
@@ -33,7 +33,12 @@
 @Immutable
 public class RuleKey implements Serializable, Comparable<RuleKey> {
 
+  /**
+   * @deprecated since 5.5, manual rule feature has been dropped
+   */
+  @Deprecated
   public static final String MANUAL_REPOSITORY_KEY = "manual";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)